### DISCUSSION TITLE
The future of dbx

### DISCUSSION THREAD

**Discussion by mitchstockdale** [databrickslabs/dbx#201] [General]
It's very exciting to see dbx make it's way into the official Databricks docs and there being an indication that the project will be [evolved into the main project](https://github.com/databrickslabs/dbx/pull/186#issuecomment-1090342619). 

With that, will the project remain open source and accept community contribution? Will anything change from the current agreements in this regard?

Cheers

**Comment by mitchstockdale** [comment]
Hey @renardeinside and @mshtelma, are you able to provide some more details and guidance on the above?

In addition, given the discussions and planning internally at Databricks for dbx, is there a product roadmap, product backlog, or something similar for contributors to view and work from?

**Comment by renardeinside** [comment]
Hi @mitchstockdale , 

there are some internal plans for the future development of dbx, but we don't have a full decision yet. As of the moment, dbx remains an OSS tool and there is no visible sign or a valid reason for dbx to become closed-source product. 

Regarding roadmap and backlog - the roadmap is defined for the two upcoming releases (0.5.0 and 0.5.1), and the backlog is already formed. Please visit [this page](https://github.com/orgs/databrickslabs/projects/2) for details.

**Reply by mitchstockdale** [reply]
Thanks @renardeinside.

I can't access the linked page, nor can I find the backlog in the projects section:

![Screen Shot 2022-05-11 at 8 48 12 am](https://user-images.githubusercontent.com/29389101/167736468-ce818d24-dbcb-4f81-b289-19224e0ac298.png)

**Comment by renardeinside** [comment]
hi @mitchstockdale, 
please try again - I've made the project page public
